src/MeetingsSDKAdapter/controls/RosterControl.test.js

```diff
@@ -0,0 +1,74 @@
+import MeetingSDKAdapter, {ROSTER_CONTROL} from '../../MeetingsSDKAdapter';
+import createMockSDK from '../../mockSdk';
+
+describe('Roster Control', () => {
+  let meeting;
+  let meetingID;
+  let meetingSDKAdapter;
+  let mockSDK;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    meetingID = 'meetingID';
+    meetingSDKAdapter = new MeetingSDKAdapter(mockSDK);
+    meeting = {
+      ID: meetingID,
+      localAudio: {
+        stream: null,
+        permission: null,
+      },
+      localShare: {
+        stream: null,
+      },
+      localVideo: {
+        stream: null,
+        permission: null,
+      },
+      remoteAudio: null,
+      remoteVideo: null,
+      remoteShare: null,
+      showRoster: null,
+      title: 'my meeting',
+      cameraID: null,
+      microphoneID: null,
+      speakerID: null,
+    };
+    meetingSDKAdapter.meetings[meetingID] = meeting;
+  });
+
+  afterEach(() => {
+    meeting = null;
+    mockSDK = null;
+    meetingSDKAdapter = null;
+    meetingID = null;
+  });
+
+  describe('display()', () => {
+    test('returns the display data of a meeting control in a proper shape', (done) => {
+      meetingSDKAdapter.meetingControls[ROSTER_CONTROL].display(meetingID)
+        .subscribe((dataDisplay) => {
+          try {
+            expect(dataDisplay).toMatchObject({
+              ID: 'member-roster',
+              icon: 'participant-list_28',
+              tooltip: 'Show participants panel',
+              state: 'inactive',
+              text: 'Participants',
+            });
+            done();
+          } catch (error) {
+            done(error);
```

Comment:
not needed, the test will fail anyway if `done()` was not called in a reasonable time.